### PR TITLE
Remove interface name from alt name if exist

### PR DESCRIPTION
**Pull request by SchSeba**
Fixes: https://github.com/k8snetworkplumbingwg/sriov-cni/issues/250

### CODE REVIEW

File: pkg/sriov/sriov.go [hunk 1]
```diff
@@ -90,8 +90,26 @@ func (s *sriovManager) SetupVF(conf *sriovtypes.NetConf, podifName string, netns
...
-	logging.Debug("3. Change netns",
+	// 3. Remove alt name from the nic
+	logging.Debug("3. Remove interface original name from alt names",
+		"func", "SetupVF",
+		"linkObj", linkObj,
+		"OriginalLinkName", linkName,
+		"tempName", tempName)
+	linkObj, err = s.nLink.LinkByName(tempName)
+	if err != nil {
+		return fmt.Errorf("error getting VF netdevice with name %s", tempName)
```
Comment (zeeke) [inline pkg/sriov/sriov.go:101]: wrap the original error
```suggestion
		return fmt.Errorf("error getting VF netdevice with name %s: %w", tempName, err)
```

**Comment by zeeke** [comment]
Only one minor comment. PR looks good to me

File: pkg/sriov/sriov.go (hunk 1 above)
Comment (SchSeba) [reply pkg/sriov/sriov.go]: done thanks! :)

**Comment by SchSeba** [comment]
Thanks @zeeke done

**Comment by SchSeba** [comment]
Hi @zeeke @e0ne @Eoghan1232 when the time allows please review this PR thanks :)

**Comment by Eoghan1232** [comment]
Hi @SchSeba I am okay with the change, but not sure why it's an issue. 
for example, other cni's use alt-name as a way to restore the original VF name
I think host-device cni uses this logic
I know sriov-cni caches the cmdAdd anyways, so we can restore from cache, thus I am okay with this change.

**Comment by SchSeba** [comment]
@Eoghan1232 thanks for the comment!

the reason we need this change (it's better explained in the issue) is that we are not able to return the vf to the host network namespace with is original name because it contains the same name in alt-name.

btw if host-cni use this feature it also needs to remove the alt-name before it restore it or it will not work

**Review by zeeke — APPROVED** [review]
LGTM

**Pull request merged**